.github/workflows/inshpect.yml

```
@@ -0,0 +1,40 @@
+name: Inshpect
+
+on:
+  pull_request:
+
+jobs:
+  doc:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Install tools
+        run: |
+          sudo apt update
+          sudo apt install -y fd-find
+          curl --output /tmp/dasel \
+            --location https://github.com/TomWright/dasel/releases/download/v2.7.0/dasel_linux_amd64
+          sudo mv /tmp/dasel /usr/local/bin/dasel
+          sudo chmod +x /usr/local/bin/dasel
+          pushd /tmp
+          curl --output inshpect \
+            --location https://raw.githubusercontent.com/msimberg/inshpect/d31ba8ed7e636c57cb9085ed3a8450d998e6b8a5/inshpect
+          echo "4b891a178fa99ac800b9b92acc0596d72f640ca1ae6de7f35ac4b3785dcd7af9 inshpect" | sha256sum --check
+          chmod +x inshpect
+          mv inshpect /usr/local/bin
+          popd
+          pushd /tmp
+          curl --output ripgrep.tar.gz \
+            --location https://github.com/BurntSushi/ripgrep/releases/download/13.0.0/ripgrep-13.0.0-x86_64-unknown-linux-musl.tar.gz
+          echo "ee4e0751ab108b6da4f47c52da187d5177dc371f0f512a7caaec5434e711c091 ripgrep.tar.gz" | sha256sum --check
+          tar --extract --strip-components 1 --file ripgrep.tar.gz
+          mv rg /usr/local/bin
+          popd
```

Just to improve readability a bit:
```suggestion

          curl --output /tmp/dasel \
            --location https://github.com/TomWright/dasel/releases/download/v2.7.0/dasel_linux_amd64
          sudo mv /tmp/dasel /usr/local/bin/dasel
          sudo chmod +x /usr/local/bin/dasel

          pushd /tmp
          curl --output inshpect \
            --location https://raw.githubusercontent.com/msimberg/inshpect/d31ba8ed7e636c57cb9085ed3a8450d998e6b8a5/inshpect
          echo "4b891a178fa99ac800b9b92acc0596d72f640ca1ae6de7f35ac4b3785dcd7af9 inshpect" | sha256sum --check
          chmod +x inshpect
          mv inshpect /usr/local/bin
          popd

          pushd /tmp
          curl --output ripgrep.tar.gz \
            --location https://github.com/BurntSushi/ripgrep/releases/download/13.0.0/ripgrep-13.0.0-x86_64-unknown-linux-musl.tar.gz
          echo "ee4e0751ab108b6da4f47c52da187d5177dc371f0f512a7caaec5434e711c091 ripgrep.tar.gz" | sha256sum --check
          tar --extract --strip-components 1 --file ripgrep.tar.gz
          mv rg /usr/local/bin
          popd
```